temporaldata/data.py

```
@@ -411,59 +412,14 @@ def load(cls, path: Union[Path, str], lazy: bool = True) -> Data:
-            if key.endswith("_domain"):
-                # domains are not split
-                assert isinstance(getattr(self, key), Interval)
-                continue
-            obj = getattr(self, key)
-            if isinstance(
-                obj, (Data, RegularTimeSeries, IrregularTimeSeries, Interval)
-            ):
-                obj.add_split_mask(name, interval)
-
-    def _check_for_data_leakage(self, name):
+    def _check_for_data_leakage(self, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)